### PR TITLE
Default darkMode to use prefers-color-scheme

### PR DESCRIPTION
**Pull request by carumusan**
If localStorage is not set then use the OS setting for dark mode.

### CODE REVIEW

File: docs/.vuepress/theme/Layout.vue [hunk 1]
```diff
@@ -36,9 +36,17 @@
     },
     mounted () {
       window.addEventListener('scroll', this.onScroll)
-      if (localStorage) {
-        this.darkMode = localStorage.getItem('dark-mode') || false
+      if (localStorage.getItem('dark-mode') !== null) {
+        this.darkMode = localStorage.getItem('dark-mode') === "true" 
       }
+      else {
+        this.darkMode = matchMedia('(prefers-color-scheme: dark)').matches
```
Comment (samajammin) [inline docs/.vuepress/theme/Layout.vue]: Please use clarify with `window.matchMedia` here for consistency.

File: docs/.vuepress/theme/Layout.vue [hunk 2]
```diff
@@ -36,9 +36,17 @@
...
+      }
+      window.matchMedia('(prefers-color-scheme: dark)').addListener(({ matches }) => {
```
Comment (samajammin) [inline docs/.vuepress/theme/Layout.vue]: Is this necessary? What's the purpose?

Comment (carumusan) [reply docs/.vuepress/theme/Layout.vue]: If you change the color scheme the website will dynamically change with it. It's not superfluous.

**Comment by samajammin** [comment]
@carumusan just for future reference, please open PRs against the `dev` branch. I'm working on https://github.com/ethereum/ethereum-org-website/issues/195 to help clarify this.

**Pull request merged**